modules/react-components/src/components/avatar/avatar.tsx

```diff
@@ -201,6 +204,27 @@ export const Avatar: FunctionComponent<PropsWithChildren<AvatarPropsInterface>>
         }
     );
 
+    /**
+     * Check if `image` is a valid image URL.
+     */
+    useEffect(() => {
+        if (image) {
+            if (React.isValidElement(image)) {
+                setIsImageValidUrl(false);
+            } else {
+                FormValidation.imageUrl(image as string).then((response) => {
```

Comment:
`joi` sends a XHR GET request to validate the image and this causes CORS issues[1]. Use the `isValidImageURL` util method [2].

[1] https://github.com/wso2/product-is/issues/9378
[2] https://github.com/wso2/identity-apps/blob/master/modules/core/src/utils/image-utils.ts#L53

Reply:
Fied by 3699ebd2